drush/Commands/SanitizationCommands.php

```diff
@@ -49,15 +49,21 @@ public function sanitizeEntities($result, CommandData $commandData) {
       return;
     }
     $types = array_keys($this->entityTypeManager->getDefinitions());
-    // This is a bit of a hack, but content moderation needs to go dead last
-    // so it can clean up after all the other stuff has been deleted.
-    $idx = array_search('content_moderation_state', $types);
-    if ($idx !== -1) {
-      unset($types[$idx]);
-      $types[] = 'content_moderation_state';
-    }
+    foreach ($types as $idx => $type) {
+
+      // Deleting menu link content sometimes results in rendering issues with
+      // super-sanitized databases, so excluding it from sanitization.
+      if ($type == 'menu_link_content') {
+        unset($types[$idx]);
```

Comment:
Does this actually accomplish the goal? Looks to me like the default entity sanitizer gets picked and its sanitize() still runs.